src/test/resources/features/selfEmployment/screenshots/SelfEmploymentAccuralUseMaximumTradingAllowanceJourney.feature

```diff
@@ -0,0 +1,152 @@
+@Screenshots
```

Comment:
Does this tag need to be removed from the 3 features files before we merge?

Reply:
No because the screenshot runner isn't used in the pipeline, so these features won't run.